### PR TITLE
Created roots_of_quadratic.py

### PR DESCRIPTION
**Pull request by githubsharat**
Wrote a python code that finds out roots of a quadratic equation. I am learning open source contribution. Please give me any feedback if possible. Thank you.

### CODE REVIEW

File: scripts/roots_of_quadratic.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#roots of quadratic equation in the form of ax^2 + bx + c = 0
```
Comment (GiovanniBruno22) [inline scripts/roots_of_quadratic.py]: Trivial: Could you move this comment inside the function and add it as a docstring instead?

File: scripts/roots_of_quadratic.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+def roots(a,b,c):
+    x1 = (-b + ((b**2 - 4*a*c)**0.5))/(2*a)
+    x2 = (-b - ((b**2 - 4*a*c)**0.5))/(2*a)
+    print ("The roots are:", x1, " and ", x2)
+
+a,b,c = input("Enter the values of a, b, and c (seperated by commas) for the equation ax^2 + bx + c = 0 : ").split(',')
```
Comment (GiovanniBruno22) [inline scripts/roots_of_quadratic.py]: Could you move line 7 and 8 inside a main function and call it using:
```
if __name__ == '__main__':
        main()
```
To stay consistent with the rest of the scripts in the repo?

Everything else looks good to me!

**Review by GiovanniBruno22 — CHANGES_REQUESTED** [review]

**Comment by githubsharat** [comment]
I am sorry I did not perform the changes yet. I am new to GitHub. I will correct my code soon.

**Comment by githubsharat** [comment]
I just made the commit: 5ecaaad2f2948b670e6150ae2407d3d63a58f6b1

Included all the suggested changes. I am sorry again about the earlier accidental resolve. Please suggest me any further feedback if you would like. Thank you very much!

**Review by GiovanniBruno22 — APPROVED** [review]
This looks good to me! thank you so much for your contribution @githubsharat !

**Comment by GiovanniBruno22** [comment]
Merging this in!

**Comment by githubsharat** [comment]
> This looks good to me! thank you so much for your contribution @githubsharat !

It’s my pleasure. Thank you very much for your feedback and opportunity to learn open-source contribution! @GiovanniBruno22

**Pull request merged**